pandas/core/series.py

```diff
@@ -2850,7 +2853,7 @@ def map(self, arg, na_action=None):
         Examples
```

Comment:
Can you add a `See Also` section? I think at least `apply` and `applymap` are related. May be the Python `map` built-in could be linked too?